packages/components/src/components/buttons/Button/Button.tsx

```diff
@@ -112,6 +117,7 @@ export const Button = ({
             iconAlignment={iconAlignment}
             disabled={isDisabled || isLoading}
             isFullWidth={isFullWidth}
+            type="button"
```

Comment:
`type` is among props, but it is hardcoded here. I think `button` should just be the default value.

Reply:
14756c45a5